config/assets/default.js

```diff
@@ -1,74 +1,4 @@
 module.exports = {
-  client: {
-    lib: {
-      // Load Angular-UI-Bootstrap module templates for these modules:
-      uibModuleTemplates: [
-        'datepicker',
-        'modal',
-        'popover',
-        'progressbar',
-        'tabs',
-        'tooltip',
-        'typeahead',
-      ],
-      js: [
-        // Non minified versions
-        'node_modules/jquery/dist/jquery.js',
-        'node_modules/angular/angular.js',
-        'node_modules/angular-aria/angular-aria.js',
-        'node_modules/angular-resource/angular-resource.js',
-        'node_modules/angular-animate/angular-animate.js',
-        'node_modules/angular-touch/angular-touch.js',
-        'node_modules/angular-sanitize/angular-sanitize.js',
-        'node_modules/angular-message-format/angular-message-format.js',
-        'node_modules/angulartics/src/angulartics.js',
-        'node_modules/angular-ui-router/release/angular-ui-router.js',
-        'node_modules/angular-ui-bootstrap/src/buttons/buttons.js',
-        'node_modules/angular-ui-bootstrap/src/collapse/collapse.js',
-        'node_modules/angular-ui-bootstrap/src/dateparser/dateparser.js',
-        'node_modules/angular-ui-bootstrap/src/datepicker/datepicker.js',
-        'node_modules/angular-ui-bootstrap/src/debounce/debounce.js',
-        'node_modules/angular-ui-bootstrap/src/dropdown/dropdown.js',
-        'node_modules/angular-ui-bootstrap/src/isClass/isClass.js',
-        'node_modules/angular-ui-bootstrap/src/modal/modal.js',
-        'node_modules/angular-ui-bootstrap/src/multiMap/multiMap.js',
-        'node_modules/angular-ui-bootstrap/src/popover/popover.js',
-        'node_modules/angular-ui-bootstrap/src/position/position.js',
-        'node_modules/angular-ui-bootstrap/src/progressbar/progressbar.js',
-        'node_modules/angular-ui-bootstrap/src/stackedMap/stackedMap.js',
-        'node_modules/angular-ui-bootstrap/src/tabs/tabs.js',
-        'node_modules/angular-ui-bootstrap/src/tooltip/tooltip.js',
-        'node_modules/angular-ui-bootstrap/src/typeahead/typeahead.js',
-        'node_modules/moment/moment.js',
-        'node_modules/angular-moment/angular-moment.js',
-        'node_modules/medium-editor/dist/js/medium-editor.js',
-        'node_modules/leaflet/dist/leaflet-src.js',
-        'node_modules/angular-simple-logger/dist/angular-simple-logger.js', // Required by angular-leaflet-directive
-        'node_modules/ui-leaflet/dist/ui-leaflet.js',
-        'node_modules/leaflet-active-area/src/leaflet.activearea.js',
-        'node_modules/angular-waypoints/dist/angular-waypoints.all.js',
-        'node_modules/ng-file-upload/dist/ng-file-upload.js',
-        'node_modules/chosen-js/chosen.jquery.js',
-        'node_modules/angular-chosen-localytics/dist/angular-chosen.js',
-        'node_modules/angular-loading-bar/build/loading-bar.js',
-        'node_modules/angular-trustpass/dist/tr-trustpass.js',
-        'node_modules/mailcheck/src/mailcheck.js',
```

Comment:
on signup, i got the error

```
TypeError
el.mailcheck is not a function
Call Stack
 link/<
  node_modules/angular-mailcheck/angular-mailcheck.js:59:16
 dispatch
  node_modules/jquery/dist/jquery.js:5237:27
 add/elemData.handle
  node_modules/jquery/dist/jquery.js:5044:28
```
could it be related to this change?

Reply:
Ah yes, good point! I had meant to go through these one by one making sure I imported them elsewhere, but ended up only doing the angular modules specifically...

Reply:
Ok, fixed. Just missing the `mailcheck` import above. This pre-dependency-aware libraries are painful!